corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -257,7 +257,7 @@ def can_use_restore_as(request):
     return (
         request.couch_user.can_edit_commcare_users() and
         has_privilege(request, privileges.DATA_CLEANUP) and
```

Comment:
seems fine to me, unless there is a reason that was put there in the first place. seems odd that we had two conditions, both of which really just check if the domain is on the pro plan